packages/dds/tree/src/test/feature-libraries/modular-schema/comparison.spec.ts

```diff
@@ -366,25 +304,15 @@ describe("Schema Comparison", () => {
 		});
 
 		it("Validate the ordering when the identifiers are different", () => {
```

Comment:
I believe this test used to be demonstrating g a bug (should not have been passing) related to handling of any. I have replaced it with a test that does what the description of the tests sounds like it should be testing based on my interpretation.